src/main/java/org/calrissian/flowmix/api/aggregator/JavascriptFunctionAggregator.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2015 Calrissian.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.calrissian.flowmix.api.aggregator;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.script.Invocable;
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+
+/**
+ *
+ * Awesome Javascript Function Aggregator. This is going to move somewhere over the rainbow
+ * 
+ * @author Miguel A. Fuentes Buchholtz
+ * @param <T> Aggregation result type
+ * @param <F> Field type
+ */
+@Deprecated
```

Comment:
Same about the deprecation. This isn't something that should be committed to begin with. If you are interested in putting this in a contrib repo, I think it should go there instead of core.